beacon/goclient/aggregator.go

```diff
@@ -54,7 +54,7 @@ func (gc *GoClient) SubmitAggregateSelectionProof(
 		AttestationDataRoot: root,
 		CommitteeIndex:      committeeIndex,
 	})
-	recordMultiClientRequest(ctx, gc.log, "AggregateAttestation", http.MethodGet, time.Since(aggDataReqStart), err)
+	recordRequest(ctx, gc.log, "AggregateAttestation", http.MethodGet, gc.multiClient.Name(), true, time.Since(aggDataReqStart), err)
```

Comment:
```suggestion
	recordRequest(ctx, gc.log, "AggregateAttestation", gc.multiClient.Address(), http.MethodGet, true, time.Since(aggDataReqStart), err)
```

The order seems to be wrong. Also, should we use `gc.multiClient.Address()` instead?
```go
func recordRequest(
	ctx context.Context,
	logger *zap.Logger,
	routeName, clientAddr, httpMethod string,
	maybeFallback bool,
	duration time.Duration,
	err error,
)
```

Reply:
Nit: Maybe we could also change the address type from `string` to `interface { Address() }`, just because several strings are error-prone